src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -217,7 +215,7 @@ class AddressValidationModal extends React.Component {
         }
         id="address-validation-warning"
         onClose={resetDataAndCloseModal}
-        visible={isAddressValidationModalVisible}
+        visible
```

Comment:
The child `Modal` will always be visible because this entire `AddressValidationModal` will be only added to the component tree when it needs to be shown.